### DISCUSSION TITLE
Is there a way to set middle mouse click to close multiple parent tabs at once?

### DISCUSSION THREAD

**Discussion by Yeove** [mbnuqw/sidebery#985] [Q&A]
![Image explanation](https://user-images.githubusercontent.com/27574845/224513521-c36ef1ae-cc95-49fd-8897-fc61db705eb3.png)

I've tried looking for an option to do this in settings, but had no luck finding a way to set this.

**Comment by megclaypool** [comment]
I would love to be able to use a keyboard shortcut to close multiple selected tabs... I keep trying to use command-w and then being surprised when only one tab closes 😆
